.vscode/tasks.json

```diff
@@ -62,6 +62,34 @@
         }
       }
     },
+    {
+      "label": "Compile and Bundle",
+      "group": {
+        "kind": "build",
+        "isDefault": false
+      },
+      "dependsOn": ["Compile"],
+      "command": "npm",
+      "type": "shell",
+      "presentation": {
+        "focus": false,
+        "panel": "dedicated"
+      },
+      "args": ["run", "vscode:bundle:extension"],
```

Comment:
I know it's hard with the pace of change, but FYI this is about to be `vscode:bundle` in the wireit branch that Cristi's reviewing for me.